### PR TITLE
 [dbunit test] remove dbunit dependency

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Finally remove dbunit from extends

Before
----------------------------------------
Discontinued package in use

After
----------------------------------------
dbunit gone

Technical Details
----------------------------------------

Comments
----------------------------------------

### CODE REVIEW

**Comment by seamuslee001** [comment]
well that worked @eileenmcnaughton

**Comment by eileenmcnaughton** [comment]
@seamuslee001 so you think this is all good / resolves the package? If so I'll fix up the pr template & you can merge

**Comment by seamuslee001** [comment]
@eileenmcnaughton I think so but not 100% sure tho i would say likely yes

File: tests/phpunit/CiviTest/CiviUnitTestCase.php [hunk 1]
```diff
@@ -244,7 +245,6 @@ public static function getDBName() {
    * @return PHPUnit_Extensions_Database_DB_IDatabaseConnection connection
```
Comment (seamuslee001) [inline tests/phpunit/CiviTest/CiviUnitTestCase.php]: @eileenmcnaughton this may need to be cleaned up

**Comment by eileenmcnaughton** [comment]
@seamuslee001 do you want to merge #14321 (has an unrelated fail but passed) & I'll rebase this & fix the comment & you can mull it until Monday or so

**Comment by seamuslee001** [comment]
Done

**Comment by seamuslee001** [comment]
This only affects tests and Jenkins has oked it, also the time taken was within normal range merging

**Pull request merged**